src/containers/Login.vue

```diff
@@ -310,6 +258,18 @@ export default Vue.extend({
 <style scoped lang="scss">
 @import '@/assets/scss/_variables.scss';
 
+.top-sec {
+  background-image: url('../assets/images/landing_new/top_rect.svg');
```

Comment:
Can you use the `~@` notation on this url just in case this file ever gets moved? There are some examples of `background-image: url(~@/...)` elsewhere in the codebase.